testnet/stacks-node/conf/puppetnet-conf.toml

```diff
@@ -0,0 +1,35 @@
+[node]
+name = "helium-node"
+rpc_bind = "127.0.0.1:20443"
+p2p_bind = "127.0.0.1:20444"
+use_test_genesis_chainstate = true
+enable_puppet_mode = true
```

Comment:
Could you add an example of the `puppet_bind` config option in this file?  Thanks!

Reply:
Added.